verifier/server/static/qr.js

```diff
@@ -0,0 +1,37 @@
+window.onload = () => {
+  const url = new URL(document.URL);
+  const example = url.searchParams.get("example") || 'kyc_age';
+  const qrUrl = `${url.origin}/api/sign-in?example=${encodeURIComponent(example)}`;
+  console.log("qrUrl:", qrUrl);
+  
+  const qrCodeEl = document.querySelector('#qrcode');
+  fetch(qrUrl)
+    .then((r) => {
+      return Promise.all([
+        r.status, 
```

Comment:
I added `status` here and some simple error handling below. Without this it shows the error as a QR. The rest is the same as before.